bin/import_mods_files

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env ruby
+
+def validate_hostname!(hostname)
+  return if hostname
+  usage
+  $stderr.puts 'Please provide the hostname to import to.'
+  exit(1)
+end
+
+def validate_mods_directory!(mods_directory)
+  if mods_directory
+    unless File.exist?(mods_directory)
+      $stderr.puts "Directory doesn't exist #{mods_directory}"
+      exit(1)
+    end
+
+    unless File.directory?(mods_directory)
+      $stderr.puts "#{mods_directory} is not a directory"
+      exit(1)
+    end
+  else
+    usage
+    $stderr.puts 'Please provide the a file path you want to import.'
+    exit(1)
+  end
+end
+
+def validate_imagepath!(imagepath)
+  return if imagepath
+  usage
+  $stderr.puts 'Image directory was left blank. No images will be ingested'
+  exit(1)
+end
+
+def load_rails
+  puts 'Loading environment...'
+  require File.expand_path('../../config/environment', __FILE__)
+  require 'importer'
+  puts 'Starting import...'
+end
+
+
+def main(hostname, mods_directory, imagepath)
+  validate_hostname!(hostname)
+  validate_mods_directory!(mods_directory)
+  validate_imagepath!(imagepath)
+  load_rails
+
+  Account.use_account!(hostname)
+  size = Importer::ModsImporter.new(imagepath, mods_directory).import_all
```

Comment:
This is in the `Importer` namespace rather than the `Stanford` one because importing MODS records is ostensibly useful outside the context of Stanford (even though our MODS import rules are probably Stanford-scented, but meh)?